ulib/LowStar.Monotonic.Buffer.fsti

```diff
@@ -1026,6 +1061,10 @@ val modifies_liveness_insensitive_mreference
 : Lemma
   (requires (modifies (loc_union l1 l2) h h' /\ loc_disjoint l1 (loc_mreference x) /\ address_liveness_insensitive_locs `loc_includes` l2 /\ h `HS.contains` x))
   (ensures (h' `HS.contains` x))
+  [SMTPatOr [
+    [SMTPat (h `HS.contains` x); SMTPat (modifies (loc_union l1 l2) h h');];
```

Comment:
same as above re goal directedness.

Also the precondition of this lemma is really long.
1 conjunct per line would be nice